packages/ionio/test/e2e/transferWithKey.test.ts

```diff
@@ -21,24 +22,41 @@ describe('TransferWithKey', () => {
       network,
       ecc
     );
-    const response = await faucetComplex(contract.address, 0.0001);
+    // re-using script's pubkey for the blinding one
+    const confidentialAddress = address.toConfidential(
+      contract.address,
```

Comment:
Isn't this a confidential address already?

Reply:
nope, we never pass a blinding private key to it inside `Contract` class